Clients/Xamarin.Interactive.Client.Web/Xamarin.Interactive.Client.Web.csproj

```diff
@@ -12,15 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="Mono.Options" Version="5.3.0.1" />
-    <PackageReference Include="Microsoft.AspNetCore" Version="2.0.2" />
-    <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.0.2" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.0.3" />
-    <PackageReference Include="Microsoft.AspNetCore.SpaServices" Version="2.0.3" />
-    <PackageReference Include="Microsoft.AspNetCore.SignalR" Version="1.0.0-preview2-30131" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="2.0.2" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />

```

Comment:
Why don't we need/want a version here?

Reply:
Because this package is a new runtime metapackage and is pinned to the .NET Core in use: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/metapackage-app?view=aspnetcore-2.1

Reply:
Also FWIW, there _is_ a version behind here. It's just implied. It even will show up as item metadata after MSBuild has evaluated it since it will come in from the SDK. It's actually pretty clever. You can see this because it ends up in our dependencies.json/md.